main.go

```diff
@@ -175,11 +180,38 @@ func handle(w http.ResponseWriter, r *http.Request) {
 		defer f.Close()
 		http.ServeContent(w, r, "main.wasm", time.Now(), f)
 		return
+
+	case "__wait":
+		waitForUpdate(w, r)
+		return
+	case "__notify":
+		notifyWaiters(w, r)
+		return
 	}
 
 	http.ServeFile(w, r, filepath.Join(".", r.URL.Path))
 }
 
+func waitForUpdate(w http.ResponseWriter, r *http.Request) {
+	// Channels are many-writers-single-reader device.
+	// so wait-ers are the writers for us...
+
+	waitChannel <- struct{}{}
+	http.ServeContent(w, r, "", time.Now(), bytes.NewReader(nil))
+}
+
+func notifyWaiters(w http.ResponseWriter, r *http.Request) {
+	for {
+		select {
+		case <-waitChannel:
+			// dump message
```

Comment:
Is this a TODO?

Reply:
comment explaining the other-wise empty block.

Reply:
This doesn't explain what this program does or what we do. Just not having a comment is fine.